docs/DeveloperGuide.md

```diff
@@ -394,20 +393,20 @@ Use case ends.
 
 **MSS**
```

Comment:
Similar to deleteMember. Might be better to indicate that the list is the filtered / currently displayed list of members. (As opposed to full list of members)

Reply:
Okay